tests/test_records.py

```diff
@@ -81,6 +85,51 @@ def test_parse_wheel_record_drop_superfulous():
     assert r1.size is None
 
 
+@pytest.mark.parametrize(
+    ("record", "data", "expected"),
```

Comment:
Instead of constructing the `Record` in the decorator's input, can we move `Record.from_elements` into the function call? The argument/values can stay a tuple called `elements`.

This avoids assuming "`Record.from_elements` will work" for being able to run this file (which tests that `Record` class).